routes/cp.php

```diff
@@ -27,7 +27,8 @@
     Route::get('select-site/{handle}', 'SelectSiteController@select');
 
     Route::group(['namespace' => 'Structures'], function () {
-        Route::resource('navigation', 'NavigationController');
+        Route::resource('navs', 'NavigationsController');
+        Route::redirect('navigation/{any?}', redirect(str_replace_first('navigation', 'navs', Request::fullUrl()), 301));
```

Comment:
Let's make this a 302, lest we decide to change again and people are stuck with a 301. (Just remove the code)

Reply:
Ah good call. Copied that from stackoverflow 😬